.github/SUPPORT.md

```diff
@@ -0,0 +1,34 @@
+# Support
+
+Thank you for using ElixirLS! :tada:
+
+This document offers some guidelines when asking for support related to an
+issues that you might encounter when using ElixirLS.
+
+# Guidelines
+
+These are some general guidelines to follow when reaching out through any of the
+available support channels.
+
+1. **Search through the existing
```

Comment:
Shouldn't we point to elixir-sense issues as well?

Reply:
Good point, added.